profile-service/src/main/java/com/transformuk/hee/tis/profile/config/FlywayConfig.java

```diff
@@ -39,7 +39,7 @@ public class FlywayConfig {
   Flyway flyway() {
     Flyway flyway = new Flyway();
     flyway.setBaselineOnMigrate(baseLineOnMigrate);
-    flyway.setLocations(migrationFilesLocations);
+    flyway.setLocations(migrationFilesLocations.split(","));
```

Comment:
Ah ha.  I got spring to do this by changing L#24 `String migrationFilesLocations` to `String[] migrationFilesLocations`.

Reply:
Ah yes. Let me raise a new PR to fix it.